wire-ios-mocktransport/Tests/Source/Push Events/MockTransportSessionTeamEventsTests.swift

```diff
@@ -1,6 +1,6 @@
 //
 // Wire
-// Copyright (C) 2024 Wire Swiss GmbH
+// Copyright (C) 2017 Wire Swiss GmbH
```

Comment:
Wait now im confused, shouldn't it be `2024`? 😁

Reply:
I intended to revert changes on the feature branch which are not mandatory.
But I agree, it shouldn't be part of this PR, reverted.